modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/StaticHtml2Generator.java

```diff
@@ -215,6 +209,31 @@ private void preparHtmlForGlobalDescription(Swagger swagger) {
         }
     }
 
+    /**
+     * Format to HTML the enums contained in every operations
+     *
+     * @param parameterList The whole parameters contained in one operation
+     * @return String | Html formated enum
+     */
+    public List<CodegenParameter> postProcessParameterEnum(List<CodegenParameter> parameterList) {
+        for(CodegenParameter parameter : parameterList) {
+            if (parameter.isEnum) {
+                parameter.enumFormatted = "";
```

Comment:
@JuH33 thanks for the enhancement. Instead of adding another property "enumFormatted", what about using vendor extensions instead?

e.g. https://github.com/swagger-api/swagger-codegen/blob/9f81aa4bfced6eddbce0131853d27592653ee3e2/modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/StaticHtml2Generator.java#L187